### DISCUSSION TITLE
Split Operation

### DISCUSSION THREAD

**Discussion by TecnoTalarn** [foldynl/QLog#352] [Q&A]
Hello everyone
I've been using QLOG for a few days now. One of the things that I see that is not well resolved is the issue of operations in Split.
When a remote station works in split, and I configure my radio for this kind of operation, when transmitting, obviously, the radio changes frequency, and QLOG detects this and also changes the indicated frequency. This causes the callsign selected in QLOG to disappear from the selection, so when I want to confirm the contact I have to retype the callsign again before saving.

I don't know if there is an option in the settings to avoid this? Not that I make many contacts in Split, but dxpeditions usually work like that, and it's always fun to do some interesting DXCC.

Cheers!

EB3AM, Jordi, from Catalonia

**Comment by foldynl** [comment]
not yet implemented. But it is present on the [TODO](https://github.com/foldynl/QLog/projects/5) list for a long time.
